lib/entities/quote/ClassicQuote.ts

```diff
@@ -161,6 +161,15 @@ export class ClassicQuote implements IQuote {
       : BigNumber.from(this.quoteData.amount);
   }
 
+  public get amountOutGasAndPortionAdjusted(): BigNumber {
+    return this.request.info.type === TradeType.EXACT_INPUT
+      // there's a possibility that quoteGasAndPortionAdjusted doesn't get populated if the flag is off
```

Comment:
This feels wrong to me, if portion is not enabled, `quoteGasAndPortionAdjusted` MUST equal `quoteGasAdjusted` in my opinion. It's still totally reasonable to use the value in `quoteGasAndPortionAdjusted`, regardless of whether portion is 0 bps, off, or any other value. Wdyt?

Reply:
+1

Reply:
for flag off i just mean when portion is not enabled (in any sense). Also I don't think quoteGasAndPortionAdjusted should ever be undefined, or any of the other methods on the DutchQuote entity like `amountOutGasAndPortionAdjusted`

Reply:
wait sorry maybe im not understanding but why does `quoteGasAndPortionAdjusted` ever have to be undefined? if portion is 0 or not on, then we just set it equal to `quoteGasAdjusted`